public/parts.gohtml

```diff
@@ -6,44 +6,96 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="">
     <meta name="author" content="Virtual Video Game Orchestra">
-    <meta name="generator" content="Jekyll v3.8.6">
 
     <title>Virtual Video Game Orchestra</title>
 
     <link rel="icon" href="images/favicons/favicon-32x32.png" sizes="32x32" type="image/png">
-    <link rel="stylesheet" type="text/css" href="css/jquery.dataTables.css">
+    <link rel="stylesheet" type="text/css" href="npm/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="npm/datatables.net-bs4/css/dataTables.bootstrap4.min.css">
+    <link rel="stylesheet" type="text/css" href="npm/datatables.net-responsive-bs4/css/responsive.bootstrap4.min.css">
+    <link rel="stylesheet" type="text/css" href="npm/@fortawesome/fontawesome-free/css/all.min.css">
     <link rel="stylesheet" type="text/css" href="css/theme.css">
     <link rel="stylesheet" type="text/css" href="css/parts.css">
+
+
     <script type="text/javascript" src="npm/jquery/dist/jquery.min.js"></script>
+    <script type="text/javascript" src="npm/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
     <script type="text/javascript" src="npm/datatables.net/js/jquery.dataTables.min.js"></script>
+    <script type="text/javascript" src="npm/datatables.net-bs4/js/dataTables.bootstrap4.min.js"></script>
+    <script type="text/javascript" src="npm/datatables.net-responsive-bs4/js/responsive.bootstrap4.min.js"></script>
+    <script type="text/javascript" src="npm/@fortawesome/fontawesome-free/js/fontawesome.min.js"></script>
```

Comment:
Yeah, all of these need to get factored out.

Reply:
I actually don't have a problem with these being in there. It's like listing out the pkg imports in a go file or the import * blocks in python.

Reply:
It's mostly because they're more or less the same everywhere.

TODO (later): Figure out how to order these / where to move them for faster page loads.